src/components/editor-page/splitter/splitter.tsx

```diff
@@ -126,7 +126,7 @@ export const Splitter: React.FC<SplitterProps> = ({
         </div>
       </ShowIf>
       <div
-        className={`${styles['splitter']} ${styles['right']} ${!showRight ? 'd-none' : ''}`}
+        className={`${styles['splitter']} ${!showRight ? 'd-none' : ''}`}
```

Comment:
Maybe change this to 
```suggestion
        className={`${styles['splitter']}${!showRight ? ' d-none' : ''}`}
```
to get rid of that trailing whitespace